RELEASE/scripts/postsool.ash

```diff
@@ -736,13 +736,24 @@ void handlePostAdventure()
 #		buffMaintain($effect[Prayer of Seshat], 5, 1, 10);
 
 		buffMaintain($effect[Singer\'s Faithful Ocelot], 280, 1, 10);
+
+		// Setup Adson Delay. We always drop Reckless Driving, but can afford to postpone dropping Intimidation for ADV delays.
+		int debuffAsdonDelay;
+
 		if(doML)
 		{
-			// Catch when we leave Smut Orc
-			if(0 < have_effect($effect[Driving Intimidatingly]))
+			// Catch when we leave Smut Orc, allow for being "side tracked" buy delay burning
+			if((0 < have_effect($effect[Driving Intimidatingly])) && (debuffAsdonDelay > 1))
```

Comment:
I don't think this will ever actually be anything but 0 since it's not persistent between runs of of postsool.

Reply:
Hmm, that is a problem. I was trying to make it so that the variable would be persistent. If I set_property then it gets overwritten each time sl_ascend is rerun correct? So if the script stops (ie. Blech House) and then immediately goes into a delay-turn due to Kramco or Vote, it will debuff.